ee/frontend/mobile-replay/mobile.types.ts

```diff
@@ -176,6 +176,21 @@ export type wireframeButton = wireframeInputBase & {
     value?: string
 }
 
+export type wireframeProgress = wireframeInputBase & {
+    inputType: 'progress'
+    /**
+     * @description This attribute specifies how much of the task that has been completed. It must be a valid floating point number between 0 and max, or between 0 and 1 if max is omitted. If there is no value attribute, the progress bar is indeterminate; this indicates that an activity is ongoing with no indication of how long it is expected to take.
+     */
+    value?: number
```

Comment:
```suggestion
    progress?: number
```

That aligns better with the Android and iOS naming, but no strong opinions.
Is this the % of the `max`? eg 0.5 is 50% of the `max`? or is the current progress of the total?

Reply:
so... in html if no max is specified then it is assumed to be 1. e.g. 0.5 would be 50%

But if you specified a max of 10 then o.5 is 5%

TLDR; it's a value relative to the max

happy to switch from value to progress

Reply:
Actually I'm going to leave it as value, since that keeps `value` having the same meaning across multiple components